db/migrate/20250516195625_remove_ite_client_foreign_key_from_claim_reconciled_claims.rb

```
@@ -0,0 +1,5 @@
+class RemoveIteClientForeignKeyFromClaimReconciledClaims < ActiveRecord::Migration[7.0]
```

RuboCop (1.75.5)

[convention] Missing top-level documentation comment for `class RemoveIteClientForeignKeyFromClaimReconciledClaims`.

(Style/Documentation)